.github/actions/dbp-charts/verify-helm/action.yml

```diff
@@ -89,14 +89,35 @@ runs:
     - name: Get commit msg
       uses: >-
         Alfresco/alfresco-build-tools/.github/actions/get-commit-message@v1.20.3
+    - name: Get a namespace to deploy on
+      id: k8sns
+      uses: >-
+        Alfresco/alfresco-build-tools/.github/actions/dbp-charts/kubernetes-valid-ns@OPSEXP-1776
```

Comment:
remember that you probably want to immediately do a new release changing those references in the release PR after merging this PR